tests/foreman/cli/test_oscap.py

```diff
@@ -24,28 +35,41 @@
     tier4,
     skip_if_bug_open,
 )
-from robottelo.datafactory import (
-    filtered_datapoint,
-    valid_data_list
-)
 from robottelo.test import CLITestCase
 
 
 @filtered_datapoint
-def valid_scapcontent_list():
-    """Returns a list of valid environment names"""
-    return[
+def invalid_id_list():
+    """Returns a list of invalid ids"""
+    return [
         gen_string('alpha'),
-        gen_string('numeric'),
-        gen_string('alphanumeric'),
-        gen_string('alphanumeric', 255)
+        gen_string('alphanumeric')
     ]
 
 
 class OpenScapTestCase(CLITestCase):
     """Tests related to the oscap cli hammer plugin"""
 
     @classmethod
+    def create_test_user_viewer_role(cls):
+        cls.login = gen_string('alpha')
+        cls.password = gen_string('alpha')
+        user = make_user({
+            'login': cls.login,
+            'password': cls.password,
+            'admin': '0'
```

Comment:
imo True/False looks better that string with numeric value, i'd rather used `'admin': False` here. Not even worth a nitpick, just fyi :)